ports/libxslt/usage

```diff
@@ -1,12 +1,10 @@
 The package libxslt is compatible with built-in CMake targets:
 
-    # xslt library
     find_package(LibXslt REQUIRED)
-    target_link_libraries(main PRIVATE LibXslt::LibXslt)
-
-    # exslt library
-    find_package(LibXslt REQUIRED)
-    target_link_libraries(main PRIVATE LibXslt::LibExslt)
+    target_link_libraries(main PRIVATE
+        LibXslt::LibXslt # xslt library
+        LibXslt::LibExslt # exslt library
+    )

```

Comment:
NO. `LibXslt::LibXslt` is to be a public link library of `LibXslt::LibExslt`, so it is not to be listed explicitly.
https://github.com/GNOME/libxslt/blob/774765ad37c31885f47ef8b964392f7f32decdaf/CMakeLists.txt#L292

Reply:
As far as I can tell, it has independent, installed headers[1] and an established public interface. Simply because the package has two libraries in it with a dependency relationship doesn't imply that the "inner" library is private. As example, consider OpenSSL's ssl and crypto libraries.

Do you have a link to documentation that directly states `LibXslt::LibXslt` is a private library?

[1] https://github.com/GNOME/libxslt/blob/774765ad37c31885f47ef8b964392f7f32decdaf/CMakeLists.txt#L143-L168

Reply:
Hm, I don't get your point. The more you argue that it is independent, the less there is a reason to list both targets.
TBH your proposal looks like the pattern for MSVC (import) libs, not for CMake targes.

> As example, consider OpenSSL's ssl and crypto libraries.

Hm, `OpenSSL::SSL` is meant to be usable without needing to explicitly list `OpenSSL::Crypto`. This *is* my libexslt usage pattern.
Another example: I can link to `Qt5::Positioning` without needing to explicitly link to `Qt5::Core`.

> Do you have a link to documentation that directly states `LibXslt::LibXslt` is a private library?

I'm saying that  `LibXslt::LibXslt` is a *public* usage requirement that is properly carried without `$<LINK_ONLY:...>` in the interface link libraries of the exported `LibXslt::LibExslt` target.
For the record, `LibXml2::LibXml2` is on the same level of dependency:
~~~
set_target_properties(LibXslt::LibExslt PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include"
  INTERFACE_LINK_LIBRARIES "LibXslt::LibXslt;LibXml2::LibXml2"
)
~~~

Reply:
PS: When I said "public", I meant it in the sense of  `PUBLIC` while building `libexslt`. When you said "private", you probably refer to the role of libxslt when using libexslt?